quickstarts/microsoft.machinelearningservices/machine-learning-job-create-or-update-job/modules/codeversion.bicep

```diff
@@ -0,0 +1,76 @@
+@description('Specifies the name of the Azure Machine Learning workspace where sweep job will be deployed')
+param workspaceName string
+
+@description('Name of the blob as it is stored in the blob container')
+param filename string = 'hello_world.py'
+
+@description('Name of the blob container')
+param containerName string = 'workspaceblobstore'
+
+@description('Azure region where resources should be deployed')
+param location string = resourceGroup().location
+
+@description('Desired name of the storage account')
+param storageAccountName string
+
+@description('Specifies the env version for sweep job.')
+param codeVersion string = '1'
+
+@description('Specifies the env for command job.')
+param codeId string = 'code'
+
+// fetching existing storage account
+resource storage 'Microsoft.Storage/storageAccounts@2021-04-01' existing = {
+  name: storageAccountName
+  // creating blob service in storage account
+  resource blobService 'blobServices' = {
+    name: 'default'
+    // creating blob container in storage account
+    resource container 'containers' = {
+      name: containerName
+      properties: {
+        publicAccess: 'Container'
+      }
+    }
+  }
+}
+
+// creating deployment script to upload script 'hello_world.py' to blob container
+resource deploymentScript 'Microsoft.Resources/deploymentScripts@2020-10-01' = {
+  name: 'deployscript-upload-blob-${uniqueString(storage::blobService::container.id)}'
```

Comment:
dependsOn: [ storage ]

you need to add this line to maintain the resource creation in a flow, if deployment script resource get ceated first without storage, it will fail as it is requiring the container.